### ISSUE TITLE
Keyboard access to hour grid

### ISSUE THREAD

**Issue by AndrewVSutherland** [AndrewVSutherland/psetpartners#71]
Currently a mouse is necessary to select the hours of availability on student.html because the checkbox grid control does not yet support keyboard access.  This is an accessibility issue that needs to be addressed.

A reasonable UI approach would be to set the focus to the upper left checkbox, allow arrows to navigate, and use shift to select (so shift down/up would function like mouse down/up).  This would not be hard to implement, just need to decide how to style the selected checkbox (which may be checked or unchecked).

**Comment by roed314** [comment]
What about just showing a light border around it (maybe light blue for contrast with the red)?

**Comment by AndrewVSutherland** [comment]
Yeah I think a blurred border might work well.  But I'm going to leave this until next week.

**Comment by AndrewVSutherland** [comment]
On a related note, the action links on the partner pane currently cannot be activated from the keyboard (they are not focusable).  This is easy to fix.

**Comment by hanryhu** [comment]
I also don't think I was able to select hours from a mobile device.  I tapped the squares, but they didn't change.

**Comment by AndrewVSutherland** [comment]
@hanryhu Mobille devices are not yet supported (specifically, IOS), see https://psetpartners.mit.edu/faq for a list of currently supported plattforms).  We hope to get it working on IOS (so that it works on an iPad) later this week.

**Comment by AndrewVSutherland** [comment]
@hanryhu iOS is now supported (since Feb, 2021 in fact), as is android (but the site is still optimized for use on a desktop or large tablet, you can now use a mobile phone, but I do not recommend it).

Keyboard access is still an open issue.  No one has actually raised this as an accessibility issue and there are some other changes I'm working on to improve the initial sign-up experience that I think are higher priority, but I will work on this next week if I find I have the time.
